mopidy_mpd/session.py

```diff
@@ -36,7 +36,11 @@ def on_line_received(self, line):
         logger.debug(
             "Response to %s: %s",
             self.connection,
-            formatting.indent(self.decode(self.terminator).join(response)),
+            formatting.indent(
+                self.decode(self.terminator).join(
+                    [r for r in response if not isinstance(r, bytes)]
```

Comment:
The square brackets on this line should be removed. It's better to pass the iterator directly to `str.join`.